jupyterlab_comments/jupyterlab_comments/handlers.py

```diff
@@ -87,3 +90,35 @@ def post(self):
         comment = self.get_argument('comment')
         self.finish(
             'Add a detached comment for a specific file (unimplemented)')
+
+class RefreshIntervalHandler(APIHandler):
+
+    def get(self):
+        try:
+            interval = refresh.get_interval()
+            self.finish(json.dumps({"interval" : interval}))
+        except Exception as e:
```

Comment:
nit: Exception too broad? what exceptions does refresh.get_interval() or self.finish may generate?

Reply:
This change will be incorporated in the next PR.